lib/internal/errors.js

```diff
@@ -28,7 +28,6 @@ function makeNodeError(Base) {
     constructor(key, ...args) {
       super(message(key, args));
       this[kCode] = key;
-      Error.captureStackTrace(this, NodeError);
```

Comment:
I checked all other occurrences as well and they are always needed besides here.